Gemfile.lock

```diff
@@ -1,20 +1,17 @@
 PATH
   remote: .
   specs:
-    amazon-pricing (0.1.76)
+    amazon-pricing (0.1.77)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    coderay (1.1.0)
+    coderay (1.1.1)
     diff-lcs (1.1.3)
+    ffi (1.9.10-java)
```

Comment:
java stuff? hmmmm... not sure i am good with that being added even though harmless.